### ISSUE TITLE
Ability to check for an attribute on a sheet without filtering that token out of the action flow entirely

### ISSUE THREAD

**Issue by iron2k2** [ironmonk88/monks-active-tiles#585]
Currently the only way to check a sheet for an attribute is to filter it, which means you must remove the token from the action flow entirely in one of two cases (Match or no match). While this works for a lot of things, being able to redirect the token to different landings based on the attribute's value would be very useful, much like redirecting a roll result, that way the token could still be sent a message or altered in some way in both cases. While a janky workaround is to create two tiles overlapping each other that have opposite filters and have them play out in tandem, this doesn't work when filtering an array (which is how you do things like detect a token's understood languages). As a result, when working with an array, there is no way to create effects that affect the token in both cases (match or no match)- Only if they positively match the filter.

Example: A player attempts to read a book that is laying open on a table. They click the book. If they understand the language, they are shown a journal stating what it says. If they don't, nothing happens, and they are confused because nothing is happening and they think something broke. If I were able to instead take that token to a landing instead of filtering it out, I could show a scrolling text only they can see indicating they don't understand the language, which would let them know why they can't read it.

**Comment by Crowguard** [comment]
That's already doable; the`Goto` landing field in Check Entity Count.

https://github.com/ironmonk88/monks-module-wiki/wiki/MATT-Check-Entity-Count#if-none-exist-go-to

**Comment by ironmonk88** [comment]
Pretty much what Crowguard said, it's already doable.
At least the example you gave can be created now.

Possibly we've misinterpreted what you're asking for though, so feel free to add additional images with the actions you're trying to perform, or ask on my Discord server to see if someone can walk you through how the achieve the effect you're looking for.

Closing the ticket for now.  Will re-open if new details come to light.
